org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java

```diff
@@ -1,7 +1,6 @@
 package org.eclipse.cpacep.util;
 
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
```

Comment:
Don't do wildcard imports.

Reply:
Hi Thomas,
This cannot be done with eclipse. When import the util classes separately, it will automatically setup the wildcard when saving.

Reply:
You can change that, of course.
`Window->Preferences->Java->Java->Code Style->Organize Imports`

Set the two values for "number of (static) imports needed for .*" to a high number.

Reply:
That's strange, it's always been working for me.
If you've made sure that it's set properly, you'll have to troubleshoot it, then..
If you can't find a solution, you could also merge the master with the new code formatter.
(But this should be an exception)